manifests/init.pp

```diff
@@ -25,32 +29,38 @@
 # @param exec_restorecon Hash of selinux::exec_restorecon resource parameters
 #
 class selinux (
+  # lint:ignore:parameter_order
```

Comment:
Why did you add the linter comment? Reordering the parameters is fine.

Reply:
I reordered as requested.

I'm not a big fan of that lint check: the arguments should be listed most important first, not pulled-from-Hiera first. It was a nice idea when it was written, though.